xbmc/messaging/helpers/DialogHelper.h

```diff
@@ -74,6 +76,27 @@ DialogResponse ShowYesNoDialogText(CVariant heading, CVariant text, CVariant noL
                                    CVariant yesLabel = CVariant(), uint32_t autoCloseTimeout = 0);
 
 /*!
+\brief This is a helper method to send a threadmessage to open a Yes/No dialog box with a cutom button
```

Comment:
(Typo: 'cutom'?)

Reply:
and wrap comments at 80 chars please

Reply:
> and wrap comments at 80 chars please

 Is this part of our coding guidelines? I'm all in for it, but nobody seems to do this...